adam/experiment/gaze_ablation.py

```diff
@@ -0,0 +1,124 @@
+from vistautils.parameters import Parameters
+from vistautils.parameters_only_entrypoint import parameters_only_entry_point
+from pegasus_wrapper import (
+    initialize_vista_pegasus_wrapper,
+    directory_for,
+    run_python_on_parameters,
+    Locator,
+    write_workflow_description,
+)
+import numpy as np
+import adam.experiment.log_experiment as log_experiment_script
+
+
+def gaze_ablation_runner_entry_point(params: Parameters) -> None:
+    """This function creates all possible gaze ablation param files within a given range"""
+    initialize_vista_pegasus_wrapper(params)
+
+    # Get the baseline experiment parameters for gaze ablation -- these are things common to all of
+    # the experiments, like:
+    #
+    #     include_image_links: true
```

Comment:
possible debugging code to remove?

Reply:
No, this is just explaining what the `gaze_ablation` namespace is expected to include. Thanks for pointing out the possibility, though.